lib/cartopy/feature.py

```diff
@@ -187,6 +187,19 @@ def geometries(self):
 
         return iter(geometries)
 
+    def with_scale(self, new_scale):
+        """
+        Return a copy of the feature with a new scale.
+
+        Args:
+
+        * new_scale:
+            The scale of dataset to be used for the new feature.
```

Comment:
It might be worth repeating the valid scales from the `__init__` docstring here, since if you are using this method you almost certainly didn't construct the object yourself but are using a pre-built instance.